tools/stf/stf_lexer.py

```diff
@@ -195,6 +196,12 @@ def t_ID(self, t):
         # print t, "pos:", t.lexpos, "col:", self.lexer.colno
         return t
 
+    @TOKEN(quoted_identifier)
+    def t_quoted_ID(self, t):
```

Comment:
Is there a better way to lex an identifier within quotes? The idea here is that any character can be used.